src/app/pages/courses/courses.module.ts

```diff
@@ -1,25 +1,18 @@
-import { BreadcrumbsModule } from './../../shared/components/breadcrumbs/breadcrumbs.module';
-import { HoursFromMinutesModule } from './../../shared/pipes/hours-from-minutes/hours-from-minutes.module';
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 import { PageContainerModule } from './../../shared/components/page-container/page-container.module';
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-
-import { CoursesContainerComponent } from './courses-container/courses-container.component';
+import { HoursFromMinutesModule } from './../../shared/pipes/hours-from-minutes/hours-from-minutes.module';
```

Comment:
for practice: try to create typescript aliases for shorter imports ;)

Reply:
thanks, great advice ;)